docs/changelog.md

```diff
@@ -235,30 +261,41 @@ description: |
     arithmetic is needed, such as for finance
   - Added `std` module for accessing standard library definitions even when a
     variable with the same name shadows/overwrites it
+    <!-- I can't find this module in the search. It should have a docs page. -->
   - Added [`array.to-dict`], [`array.reduce`], [`array.windows`] methods
   - Added `exact` argument to [`array.zip`]
   - Added [`arguments.at`] method
   - Added [`int.from-bytes`], [`int.to-bytes`], [`float.from-bytes`], and
     [`float.to-bytes`]
-  - [`calc.round`] no longer accepts negative digits (**Minor breaking change**)
+  - `digits` parameter of [`calc.round`] no longer accepts negative integers
+    (**Minor breaking change**)
   - Conversions from [`int`] to [`float`] will now error instead of saturating
     if the float is too large (**Minor breaking change**)
+    <!-- Pretty sure docs should be updated here:
```

Comment:
it is mentioned in the int constructor docs

Reply:
You mean this?

> [Raises an error if there is an attempt to produce an integer larger than the maximum 64-bit signed integer or smaller than the minimum 64-bit signed integer.](https://staging.typst.app/docs/reference/foundations/int/#constructor)

This applies to all data types, but the changelog specifically mentions `float`, so I thought that the `float` part is missing. Maybe the changelog should include all data types or something (be exclusive).

Reply:
Can you please confirm (if you already read the comment) if I linked to what you are referring to? I want to undo all changes that were applied in one way or another. This way I will be left only with stuff that needs to be addressed in the future (when I get more time).